### DISCUSSION TITLE
스타일링 관련 디테일

### DISCUSSION THREAD

**Discussion by dongrri22** [mi-gan-zi/FE-MiGanZi#124] [스타일]
배포된 사이트 기준으로 스타일링 수정이 필요해보이는 부분들 적어봅니다!
이 외에도 수정했으면 좋겠는 부분들 코멘트 달아주세요~

전체
 1. 스크롤 기능은 작동하는데, 스크롤바가 안보임
 2. 상단의 디자인 요소(시간, 아이콘) 마진 필요해 보임 
![image](https://github.com/mi-gan-zi/FE-MiGanZi/assets/44963323/490260bc-031d-408c-94ef-89d3fecba47c)
 3. 상단의 디자인 요소 배치 수정 필요해보임

메인
 4. 제목 밑의 날짜와 조회수 사이에 간격 필요해보임

생성
 5. 음악을 선택해 주세요 문구 줄바꿈 하면 좋을 것 같음 
![image](https://github.com/mi-gan-zi/FE-MiGanZi/assets/44963323/94aa4b05-e280-4725-8cbb-b2d687084e37)

로그인
 6. 로그인 페이지 가로 스크롤 없애야함 
![image](https://github.com/mi-gan-zi/FE-MiGanZi/assets/44963323/5d935f9e-53d7-46f5-8ec0-850af257179c)

마이
 7. 마이페이지의 알림설정 토글 색상 변경 필요해보임 
![image](https://github.com/mi-gan-zi/FE-MiGanZi/assets/44963323/afd99925-1ab3-4a22-876f-802471616b2b)

**Comment by pangkyu** [comment]
2번 디자인 요소 경우에 모바일 뷰에서 똑같이나오면 디자인이 중복된다고 생각합니다 
모바일인 경우에는 해당 디자인이 안보이도록 설정하는 부분을 추가하면 좋을 것 같아요

**Comment by 0gunkim** [comment]
헤더 제거, 플레이어 음악이 없을 때 텍스트 제거 -> 일단 두 디자인은 디자이너님이 오더 주신게 아니라서  그냥 뺐습니다.
